pkg/llms/dashscope/api.go

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright 2023 KubeAGI.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package dashscope
+
+import (
+	"context"
+	"errors"
+
+	"github.com/kubeagi/arcadia/pkg/llms"
+)
+
+const (
+	DashScopeChatURL = "https://dashscope.aliyuncs.com/api/v1/services/aigc/text-generation/generation"
+)
+
+type Model string
+
+const (
+	QWEN14BChat Model = "qwen-14b-chat"
+	QWEN7BChat  Model = "qwen-7b-chat"
+)
```

Comment:
Can you pls add this embedding service as well ? https://help.aliyun.com/zh/dashscope/developer-reference/text-embedding-quick-start?spm=a2c4g.11186623.0.0.52f85038Fa45wY

See https://github.com/kubeagi/arcadia/tree/main/pkg/embeddings/zhipuai 

You can open another PR if you want.

Reply:
> add this embedding service as well

sure, I will add embedding from another pr.